tests/network/probes.go

```diff
@@ -103,6 +100,9 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 				Expect(matcher.ThisVMI(vmi)()).To(matcher.HaveConditionMissingOrFalse(v1.VirtualMachineInstanceReady))
 
 				By("Starting the server inside the VMI")
+				Eventually(matcher.ThisVMI(vmi)).WithTimeout(6 * time.Minute).WithPolling(3 * time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceAgentConnected))
```

Comment:
Do you really need 6 minutes? Also it is weird that above we expect the VMI not to be ready.

Reply:
I am just questioning the assert above and the large timeout